dbt/adapters/sqlserver/__init__.py

```diff
@@ -1,6 +1,7 @@
 from dbt.adapters.sqlserver.connections import SQLServerConnectionManager
 from dbt.adapters.sqlserver.connections import SQLServerCredentials
 from dbt.adapters.sqlserver.impl import SQLServerAdapter
+from dbt.adapters.sqlserver.relation import SQLServerRelation
```

Comment:
Switch the order of these:
```suggestion
from dbt.adapters.sqlserver.relation import SQLServerRelation
from dbt.adapters.sqlserver.impl import SQLServerAdapter
```
Because `SQLServerAdapter` depends on `SQLServerRelation`, you can't initialize the former before the latter. Here's the error I get locally:
```
Encountered an error:
cannot import name 'SQLServerRelation' from partially initialized module 'dbt.adapters.sqlserver' (most likely due to a circular import) (/Users/jerco/dev/product/plugins/msft/dbt-sqlserver/dbt/adapters/sqlserver/__init__.py)
```